packages/dx-react-scheduler/src/plugins/all-day-panel.tsx

```diff
@@ -85,51 +88,88 @@ class AllDayPanelBase extends React.PureComponent<AllDayPanelProps, AllDayPanelS
         dependencies={pluginDependencies}
       >
         <Getter name="allDayElementsMeta" value={elementsMeta} />
+        <Getter name="allDayCellsData" computed={this.allDayCellsDataComputed} />
+        <Getter name="allDayPanelExists" computed={this.allDayPanelExistsComputed} />
         <Getter
           name="allDayAppointments"
           computed={this.allDayAppointmentsComputed}
         />
-        <Template name="body">
+
+        <Template name="timeTable">
           {(params: any) => (
             <TemplateConnector>
-              {({ groupOrientation, currentView }) => {
-                if (currentView.type === VIEW_TYPES.MONTH) {
-                  return <TemplatePlaceholder params={{ ...params }} />;
+              {({ currentView, groupOrientation, allDayCellsData }) => {
+                if (currentView.type === VIEW_TYPES.MONTH
+                    || groupOrientation?.(currentView.name) !== VERTICAL_GROUP_ORIENTATION) {
```

Comment:
`currentView.type` and `currentView.name`. 
These fields seems very similar.
Maybe we should rename 'currentView.name` => `currentView.orientation` or something like that?

Reply:
These fields serve different functions. `Type` is responsible for the view's type. Currently, there are three of them: 'month', 'week' and 'day'. `name`, however, is view's unique identifier. This is how we can differentiate between two or more view's of the same type (for example, there may be 5 month views)